README.md

```diff
@@ -25,32 +25,40 @@ buildscript {
 plugins {
   id "net.ltgt.apt" version "0.11"
   id "net.ltgt.errorprone" version "0.0.11"
-  id "java"
+  id "java-library"
```

Comment:
what about folks who use the standard "java" plugin? This might be confusing for that use case right?

Reply:
We should probably cover both otherwise it might give the wrong message that nullaway is not supported with the older "java" plugin

Reply:
I can add a comment, I suppose `java` is still used for some applications

Reply:
5217d2f